packages/docs/concepts/events-and-actions.yaml

```diff
@@ -24,41 +24,25 @@ _ref:
-            - A list of actions are executed sequentially by a triggered event.
-            - If an action errors, the actions that follow are skipped.
-            - Actions that are `async: true` will not executed sequentially nor stop the event if they error.
-            - Action errors can be handled by providing a list of `try` and `catch` actions to the event.
-            - Operators used in action `params` are evaluated right before the action is executed.
-            - The [`_actions`](/_actions)) operator is available for sequential actions to use the values returned from preceding actions in the chain.
-            - Actions have a `skip` field that can be used to skip action execution.
-            - The `onInit` event is triggered the first time a context is mounted and keeps the page in loading until all actions have finished.
-            - The `onInitAsync` event is triggered the first time a context is mounted and does not keep the page in loading.
-            - The `onMount` events is triggered the every time a block is mounted and keeps the block in loading until all actions have finished.
-            - The `onMountAsync` event is triggered the every time a block is mounted, after `onMount` has completed, and does not keep the block in loading.
-
-            -----------
-
             Blocks can define _events_ which the block can trigger when something happens on the page, like a button being clicked, an input's value being modified or a page being loaded. Some examples are `onClick` on a [`Button`](/Button) or `onMount` on a [`PageHeaderMenu`](/PageHeaderMenu) block.
 
             All blocks implements `onMount` and `onMountAsync` events. These are useful for triggering actions when a block is mounted. For the `onMount` event, the block only mounts when the event action chain is completed, however, for the `onMountAsync` event, the block will mount as soon as possible while the event actions completes execution.
 
-            _Actions_ are tasks that can be executed, like calling a request, linking to a new page or changing a value in state. An array of actions can be defined for a event on a block. If that event gets triggered, those actions will execute sequentially. If any actions error while executing, the actions that follow it won't be executed, however, `catch` actions chain can be defined on a event to trigger when an error in a chain of actions occurs.
+            _Actions_ are tasks that can be executed, like calling a request, linking to a new page or changing a value in state. An array of actions can be defined for an event on a block. If that event gets triggered, those actions will execute sequentially. If any actions error while executing, the actions that follow it won't be executed, however, `catch` actions chain can be defined on an event to trigger when an error in a chain of actions occurs.
```

Comment:
I think we can update this to something like this:
```
_Actions_ are tasks that can be executed, like calling a request, linking to a new page or changing a value in state. An array of actions can be defined for an event on a block. If that event gets triggered, those actions will execute sequentially. 

The chain of actions will stop executing if the action raises a break condition (what do we call this?) (eg. when a form validation does not pass) or if the action throws an unexpected error.

If a break condition was raised, the user will be shown an error message and no further actions will execute. If an unexpected error is throw the app will show an error screen and the error will be logged. To handle errors and break conditions, a `catch` action chain can be defined on the event to handle the error, as described below.
```
@Gervwyk please check and confirm the described behaviour

Reply:
Changed this to be similar to what you have suggested there, for now.

Change made in https://github.com/lowdefy/lowdefy/commit/eb614758df9c5036d24ce224142dda3c60172021